src/frontend/src/components/Header/DesktopHeader.jsx

```diff
@@ -126,7 +154,7 @@ const Header = () => {
               Home
             </Link>
           </Button>
-          <Login />
+          {username === false ? <LoggedOut /> : <Link to="/myfeeds">{username}</Link>}
```

Comment:
Same deal here, flip it: `{username ? (<Link to="/myfeeds">{username}</Link>) : <LoggedOut />}`